lens.cabal

```diff
@@ -428,6 +431,9 @@ test-suite doctests
   default-language:  Haskell2010
   x-doctest-options: --fast
 
+  if impl(ghcjs)
+    buildable: False
```

Comment:
Why is this not buildable on GHCJS?

Reply:
doctest depends on ghc, tricky with ghcjs

Reply:
Interesting. Is there an issue about this somewhere where I can read more about this?

Reply:
Not really. I don't think it's worth an issue. `doctest` doesn't work with any other compiler than GHC, neither with GHCi-less version (which GHCJS is).